vmaas_sync/metrics_cyndi.go

```diff
@@ -0,0 +1,125 @@
+package vmaas_sync //nolint:golint,stylecheck
+
+import (
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"fmt"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+const (
+	allSystemCount        = "allSystem"
+	uniqueTagsCount       = "uniqueTags"
+	systemsSapSystemCount = "systemsSapSystem"
+	systemsWithTagsCount  = "systemsWithTag"
+)
+
+var (
+	cyndiSystemsCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_systems",
+	}, []string{"type"})
+
+	cyndiTagsCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_tags_count",
+	}, []string{"type"})
+
+	tagsPerSystem = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "tags_per_system",
+	})
+)
+
+type InventoryHosts struct {
```

Comment:
InventoryHostsStats is better struct name here. Moreover you use this name in models.go as well but for something else.

Reply:
Please name it InventoryHostsStats. It's more descriptive name for it.